const anotherFunction = () => {
    return new Promise((resolve, reject) => {
        if (true) {
            resolve('Logrado!')
        } else {
            reject('No funcionÃ³.')
        }
    })
}

anotherFunction()
    .then(response => console.log(response))
    .catch(err => console.log(err));
